### PR TITLE
Documentation for CVE Visualizations feature.

### PR DESCRIPTION
**Pull request by amouat**
*DO NOT MERGE BEFORE CVE VIZ IS IN BETA*

## Type of change
Add docs for CVE Visualizations

### How should this PR be tested?
Please check the language. Feel free to click through the feature as well, you may need to enable it first by using this URL:
https://console.chainguard.dev/reports?feature.compare=true

[Direct link to preview](https://deploy-preview-1940--ornate-narwhal-088216.netlify.app/chainguard/chainguard-images/working-with-images/cve_visualizations/)

### CODE REVIEW

**Review by matthewhelmke — APPROVED** [review]
This looks good. There are a couple places where I would prefer to see a comma, but I don't think they are a grammatical must-do, just a nice-to-have.

File: content/chainguard/chainguard-images/working-with-images/cve_visualizations/index.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+tags: ["CONCEPTUAL", "CHAINGUARD IMAGES", "PRODUCT"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 020
+toc: true
+---
+
+The CVE Visualizations feature in the Chainguard Console helps teams measure the engineering,
```
Comment (angela-zhang) [inline content/chainguard/chainguard-images/working-with-images/cve_visualizations/index.md:18]: Maybe we can open or add this positioning statement somewhere: "CVE Visualizations are a single source of truth that bring transparency to image health and continuously articulate the engineering, security, and economic benefits of adopting Chainguard's guarded, minimal, zero-CVE images to your internal stakeholders."

File: content/chainguard/chainguard-images/working-with-images/cve_visualizations/index.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+security and economic benefits gained from using Chainguard Images. The primary feature is a
```
Comment (angela-zhang) [inline content/chainguard/chainguard-images/working-with-images/cve_visualizations/index.md:19]: We'll also be adding a separate tab in the Reports section for Beta with historical CVE remediation )showing the # of resolved CVEs over time per image). Maybe reword to something like "The feature provides self-service reporting for CVE comparisons counts between Chainguard Images and popular open-source alternatives, as well as historical CVE remediation metrics for your entitled Chainguard Images"

File: content/chainguard/chainguard-images/working-with-images/cve_visualizations/index.md (hunk 1 above)
Comment (amouat) [reply content/chainguard/chainguard-images/working-with-images/cve_visualizations/index.md]: I reworded a bit, but these docs are developer facing. We try to keep the language concise and focused on getting tasks done, whereas the language here is more appropriate for a blog or marketing flyer.

File: content/chainguard/chainguard-images/working-with-images/cve_visualizations/index.md (hunk 2 above)
Comment (amouat) [reply content/chainguard/chainguard-images/working-with-images/cve_visualizations/index.md]: Tried to reword a bit. "self service" seems redundant in this context and the metrics seem to be for all images now (not just entitlements)

**Comment by amouat** [comment]
I moved to draft to avoid accidental commits.

File: content/chainguard/chainguard-images/working-with-images/cve_visualizations/cve_report.png
Comment (angela-zhang) [inline content/chainguard/chainguard-images/working-with-images/cve_visualizations/cve_report.png:1]: when we update this screenshot, we should use a non-internal console login view (that doesn't show v129 and Features)

Comment (angela-zhang) [reply content/chainguard/chainguard-images/working-with-images/cve_visualizations/cve_report.png:1]: let's wait until we have the Historical CVEs tab available before we update the screenshot
